examples/ft-ui/__test__/anonymous-user.test.js

```diff
@@ -19,13 +18,12 @@ describe('examples/ft-ui', () => {
     })
 
     it('does not render the logged-in user Header link elements', async () => {
-      await expect(page).not.toMatchElement('.o-header__top-column--right a[href$="/myaccount"]', {
-        text: 'Settings & Account'
-      })
```

Comment:
**Reviewer notes:** Becuase the sign in and my account link is now the same location (previously it moved location when logged in) this test no longer makes sense because we already have a test ensuring that this element exists and its text is "Sign in"